tests/unittest/gcp_test.py

```diff
@@ -61,6 +76,19 @@
     }
 }
 
+# Mock struct to mimic GCP's API responses
+MOCK_LOGS_LIST = {
```

Comment:
Any reason we move from `FAKE_` to `MOCK_` @giovannt0?

Reply:
@tomchop I think we used to call `FAKE...` when we were instantiating objects and `MOCK...` when we were defining an API response.

Seeing the following:

```
FAKE_LOG_LIST = [
    'projects/fake-target-project/logs/GCEGuestAgent',
    'projects/fake-target-project/logs/OSConfigAgent'
]
...
MOCK_LOGS_LIST = {
    'logNames': FAKE_LOG_LIST
}
```

We should probably group that together as:

```
MOCK_LOGS_LIST = {
    'logNames': [
        'projects/fake-target-project/logs/GCEGuestAgent',
        'projects/fake-target-project/logs/OSConfigAgent'
    ]
}
```

Same for `FAKE_LOG_ENTRIES` and `MOCK_LOG_ENTRIES`.

wdyt? @hacktobeer?

Reply:
I think that was the idea, but my knowledge from 13 months ago is not super up-to-date anymore. I can work with whatever you Python experts agree on :).

Reply:
OK let's make sure to check that this is consistent during the refactor tomorrow.